core/templates/dev/head/profile/profile.html

```diff
@@ -85,7 +85,8 @@ <h2 class="oppia-profile-username-small-screen">
         Interests:
         <span ng-if="subjectInterests.length > 0"><br>
           <!-- The following tag should remain on a single line due to http://bugzilla.mozilla.org/show_bug.cgi?id=922773. -->
-          <span ng-repeat="interest in subjectInterests track by $index"><span class="oppia-profile-subject-interest"><[interest]></span></span>
+          <span ng-repeat="interest in subjectInterests track by $index">
```

Comment:
I know that Chrome has updated at least twice since that thread, and I'm assuming that Firefox has as well.  I didn't search too hard, but I assume both web browsers changed something that made the old code not work.